airflow/providers/amazon/aws/secrets/secrets_manager.py

```diff
@@ -81,6 +105,7 @@ def __init__(
         profile_name: Optional[str] = None,
         sep: str = "/",
         full_url_mode: bool = True,
+        extra_conn_words: Optional[dict] = {},
```

Comment:
[Don’t usage mutable values as function argument default](https://docs.python-guide.org/writing/gotchas/)

Reply:
Oh. YEAH. That one is so non-obvious, but so dangerous. I think that was about the only really useful Pylint check that we miss now. But I just checked that we could use https://pypi.org/project/flake8-mutable/ extension.